test/expected/insert.out

```diff
@@ -38,89 +38,103 @@ INSERT INTO "two_Partitions"("timeCustom", device_id, series_0, series_1) VALUES
- _timescaledb_internal."_hyper_1_3_chunk_two_Partitions_timeCustom_series_bool_idx" | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_3_chunk_two_Partitions_timeCustom_series_bool_idx" | i    | series_bool | boolean          | f
- _timescaledb_internal._hyper_1_4_chunk                                             | r    | timeCustom  | bigint           | t
- _timescaledb_internal._hyper_1_4_chunk                                             | r    | device_id   | text             | t
- _timescaledb_internal._hyper_1_4_chunk                                             | r    | series_0    | double precision | f
- _timescaledb_internal._hyper_1_4_chunk                                             | r    | series_1    | double precision | f
- _timescaledb_internal._hyper_1_4_chunk                                             | r    | series_2    | double precision | f
- _timescaledb_internal._hyper_1_4_chunk                                             | r    | series_bool | boolean          | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_device_id_timeCustom_idx"   | i    | device_id   | text             | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_device_id_timeCustom_idx"   | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_device_id_idx"   | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_device_id_idx"   | i    | device_id   | text             | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_idx"             | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_0_idx"    | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_0_idx"    | i    | series_0    | double precision | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_1_idx"    | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_1_idx"    | i    | series_1    | double precision | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_2_idx"    | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_2_idx"    | i    | series_2    | double precision | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_bool_idx" | i    | timeCustom  | bigint           | f
- _timescaledb_internal."_hyper_1_4_chunk_two_Partitions_timeCustom_series_bool_idx" | i    | series_bool | boolean          | f
-(76 rows)
+                                      Relation                                      | Kind |        Column        |       Column type        | Nullable 
```

Comment:
Should probably do ` test.show_columnsp('_timescaledb_internal._hyper_%');` here to avoid outputting job_stats tables.

Reply:
Fixed